core/jazz_slack-event-handler/index.js

```diff
@@ -0,0 +1,311 @@
+        transform: (body, response, resolveWithFullResponse) => {
+            return response;
+        }
+    };
+};
+
+function getAuthResponse(result) {
+    return new Promise((resolve, reject) => {
+        if (result.statusCode === 200 && result.body && result.body.data) {
+            return resolve(result.body.data.token);
+        } else {
+            logger.error("getAuthResponse failed");
+            return reject(errorHandler.throwInternalServerError("Invalid token response from API"));
+        }
+    });
+}
+
+function processRecords(event, configData, authToken) {
+    return new Promise((resolve, reject) => {
+        var processRecordPromises = [];
+        for (let i = 0; i < event.Records.length; i++) {
+            processRecordPromises.push(exportable.processRecord(event.Records[i], configData, authToken));
+        }
+        Promise.all(processRecordPromises)
+            .then((result) => { return resolve(result); })
+            .catch((error) => { return reject(error); });
+    });
+}
+
+function processRecord(record, configData, authToken) {
+    return new Promise((resolve, reject) => {
+        var sequenceNumber = record.kinesis.sequenceNumber;
+        var encodedPayload = record.kinesis.data;
+        var payload;
+        return exportable.checkInterest(encodedPayload, sequenceNumber, configData)
+            .then(result => {
+                payload = result.payload;
+                if (result.interested_event) {
+                    return exportable.processEvent(payload, configData, authToken);
+                }else {					
+					resolve({ "message": "Not an interesting event" });					
+				}
+            })
+            .then(result => {
+                exportable.handleProcessedEvents(sequenceNumber, payload);
+                return resolve(result);
+            })
+            .catch(err => {
+                exportable.handleFailedEvents(sequenceNumber, err.failure_message, payload, err.failure_code);
+                return reject(err);
+            });
+    });
+}
+
+function checkInterest(encodedPayload, sequenceNumber, configData) {
+    return new Promise((resolve, reject) => {
+        var kinesisPayload = JSON.parse(new Buffer(encodedPayload, 'base64').toString('ascii'));
+        logger.info("kinesisPayload :" + JSON.stringify(kinesisPayload));
+        if (kinesisPayload.Item.EVENT_TYPE && kinesisPayload.Item.EVENT_TYPE.S) {
+            if (configData.EVENTS.EVENT_TYPE.includes(kinesisPayload.Item.EVENT_TYPE.S) &&
+                configData.EVENTS.EVENT_NAME.includes(kinesisPayload.Item.EVENT_NAME.S)) {
+                logger.info("found " + kinesisPayload.Item.EVENT_TYPE.S + " event with sequence number: " + sequenceNumber);
+                return resolve({
+                    "interested_event": true,
+                    "payload": kinesisPayload.Item
+                });
+            } else {
+                logger.error("Not an interested event or event type");
```

Comment:
While checking something else, I found that this statement is polluting our logs. Can you make this `debug`?